pkg/cluster/cluster.go

```diff
@@ -397,6 +401,45 @@ func (os *ClusterSpec) Validate() error {
 	if s.InitMode == nil {
 		return fmt.Errorf("initMode undefined")
 	}
+
+	symbolsNameFilter := regexp.MustCompile("[_0-9]")
+	if len(s.ExternalFallbackSyncStandbys) > 0 {
+		correctSymbolsNameFilter := regexp.MustCompile("[_0-9A-Za-z]")
+		for _, standbyName := range s.ExternalFallbackSyncStandbys {
+			if trimmedStandbyName := strings.TrimSpace(standbyName); len(trimmedStandbyName) == 0 ||
+				len(correctSymbolsNameFilter.ReplaceAllString(trimmedStandbyName, "")) > 0 ||
+				strings.Contains(standbyName, "\n") ||
+				len(trimmedStandbyName) != len(standbyName) {
+				return fmt.Errorf("External fallback synchronous standby name \"%s\" has an incorrect format, it may only contain letters, numbers, and the underscore character", standbyName)
+			}
+			if filteredSymbolsOnlyName := symbolsNameFilter.ReplaceAllString(standbyName, ""); len(filteredSymbolsOnlyName) == 0 {
+				return fmt.Errorf("External fallback synchronous standby name \"%s\" has an incorrect format, every name should start with a symbol", standbyName)
+			}
+
+			if common.IsStolonName(standbyName) {
+				return fmt.Errorf("External fallback synchronous standby name \"%s\" has an incorrect format, it should not have prefix 'stolon_'", standbyName)
+			}
+			if strings.Contains(standbyName, "\n") {
+
+			}
+		}
+	}
+
+	if len(s.AdditionalReplicationSlotNames) > 0 {
+		rp := regexp.MustCompile("[_0-9a-z]+")
```

Comment:
Can't you just use https://github.com/sorintlab/stolon/blob/master/pkg/postgresql/utils.go#L328 ?